source/fab/builder.py

```diff
@@ -72,6 +72,7 @@ def entry() -> None:
                         choices=range(2, multiprocessing.cpu_count()),
                         help='Provide number of processors available for use,'
                              'default is 2 if not set.')
+    parser.add_argument('--stop-on-error', action="store_true")
```

Comment:
Will we ever need to not stop on an error? Errors tend to be pretty terminal.

Reply:
It's definitely useful to see all the errors in one output stream rather than having to keep fixing one problem at a time then re-running. With the latter you don't get an idea of how far through the fixing process you are. With the former you can fix lots of stuff at once if you like. Better still, if you can see all the classes of errors that you need to fix, you can potentially pick a better solution sooner.

Reply:
In which case would you ever want to stop on an error? Would you not prefer to view all errors as you suggest?

Reply:
Good question. I've been using it. It's useful to stop on the first error sometimes so you don't have to search the output for the first one. This flag seems likely to evolve into the more sophisticated behaviour the team's been discussing. I think this is a reasonable step in the right direction.

Reply:
I'd somehow missed the request to remove this from _user_ control, now done.